src/main/js/src/containers/Quarantine/QuarantineHistory.js

```diff
@@ -0,0 +1,188 @@
+
+import Button from '../../components/UI/Button/Button';
+import withErrorHandler from '../../hoc/withErrorHandler/withErrorHandler';
+import Page from '../../hoc/Page/Page';
+import classes from './Quarantine.module.css';
+import * as actions from '../../store/actions/index';
+import { DATE_FORMAT, LANGUAGES } from '../../common/Constants';
+import Spinner from '../../components/UI/Spinner/Spinner';
+import QuarantineNav from '../../components/Quarantine/Navigation';
+import QuarantineConfirmModal from '../../components/Quarantine/ConfirmModal';
+
+const QuarantineHistory = props => {
+  const {
+    t,
+    reviews,
+    onFetchQuarantineReviews,
+    setQuarantine,
+    confirmQuarantine,
+    closeConfirmQuarantine,
+    confirm,
+    loading,
+    error,
+  } = props;
+  const findLang = (language) => LANGUAGES.find(l => l.code === language).name;
+  const closeConfirmModal = () => closeConfirmQuarantine();
+  const hasError = !R.isNil(error);
+
+  useEffect(onFetchQuarantineReviews, [hasError]);
+
+  const showQuarantineConfirm = (id, reg_id) =>
+    confirmQuarantine(() => setQuarantine(id, reg_id, true), t('quarantine.confirmDescription'));
+
+  const showCancelQuarantineConfirm = (id, reg_id) =>
+    confirmQuarantine(() => setQuarantine(id, reg_id, false), t('quarantine.confirmCancelDescription'));
+
+  return (
+    <Page>
+      {R.isNil(error) && !R.isNil(confirm) && (
+        <QuarantineConfirmModal
+          t={t}
+          confirm={confirm.callback}
+          cancel={closeConfirmModal}
+          loading={loading}
+          description={confirm.description}
+        />
+      )}
+      <div className={classes.QuarantineMatches}>
+        <h1>
+          {t('quarantine.reviewsTitle')}
+        </h1>
+
+        <QuarantineNav t={t} />
+
+        <p>
+          {t('quarantine.reviewsDescription')}
+        </p>
+
+        <div className={classes.QuarantineList}>
+          <div/>
+          <div className={classes.ListHeader}>
+            {t('common.examLanguage')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('common.examDate')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('common.names')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('common.email')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('common.birthdate')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('common.phoneNumber')}
+          </div>
+          <div className={classes.ListHeader}>
+            {t('quarantine.status')}
+          </div>
+          <div/>
+          {reviews.map(review => (
+            <React.Fragment key={`quarantine-match-row-${review.quarantine_id}`}>
+              <div className={classes.IndicatorRow}>
+                <span>{t('common.quarantine')}</span>
+                <span>{t('common.registration')}</span>
+              </div>
```

Comment:
Wireframessa nämä taisi olla toisin päin, eli ilmoittautumisrivi alempana. En sinänsä ota kantaa siihen kumpi on parempi, mutta voisi ainakin synkata toteutuksen ja designin keskenään.